Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/LocationUpdateService.java

```diff
@@ -660,7 +809,11 @@ public void onLocationResult(LocationResult locationResult) {
                     {
                         updateStorage("LAST_KNOWN_LAT", String.valueOf(lastLatitudeValue));
                         updateStorage("LAST_KNOWN_LON", String.valueOf(lastLongitudeValue));
-                        callDriverCurrentLocationAPI(lat, lng, acc, thisLocationTimeStamp, "fused_location_callback", LocationSource.LastLocation.toString(), TriggerFunction.GoogleCallback.toString());
+                        try {
```

Comment:
why is wrapping this function with try/catch block required? Function itself has a try/catch block within

Reply:
The function `callDriverCurrentLocationAPI` throws a json exception. 
So wherever it is called it needs to be within try catch block.

Reply:
Then handle the exception within the function, no need to add throws Exception in front of function